packages/lab/src/stepper/Step.Connector.css

```diff
@@ -0,0 +1,52 @@
+.saltStepConnector {
+  --saltStep-connector-borderWidth: 2px;
+
+  grid-area: connector;
+  min-height: var(--salt-spacing-300);
+
+  transition-duration: inherit;
+  transition-timing-function: inherit;
+  transition-property: opacity, min-height;
+}
+
+.saltStepper-horizontal .saltStepConnector {
+  /* Distance between Icon and Connector */
+  --saltStep-connector-horizontal-gap: var(--salt-spacing-100);
+
+  position: absolute;
+  top: calc(var(--icon-size) / 2 - var(--saltStep-connector-borderWidth));
+  left: calc(50% + calc(var(--icon-size) / 2 + var(--saltStep-connector-horizontal-gap)));
+  right: calc(-50% + calc(var(--icon-size) / 2 + var(--saltStep-connector-horizontal-gap)));
+
+  border-top-width: var(--saltStep-connector-borderWidth);
+  border-top-style: var(--salt-track-borderStyle-incomplete);
+  border-top-color: var(--salt-track-borderColor);
+}
+
+.saltStepper-horizontal .saltStep-stage-completed .saltStepConnector {
+  border-top-style: var(--salt-track-borderStyle-complete);
+}
+
+.saltStepper-vertical .saltStepConnector {
+  height: 100%;
+  min-height: 24px;
```

Comment:
where does this 24 come from? it should probably be a token... probably something like `--salt-size-base` or base+x

Reply:
Well, in the design it's not a token... In the design it goes 12, 24, 36, 48 based on the densities. I have it wrong for sure, but I don't know with what variable to replace it with 😔